plugin/src/main/groovy/com/novoda/buildproperties/internal/FilePropertiesEntries.groovy

```diff
@@ -44,6 +44,13 @@ class FilePropertiesEntries implements Entries {
         provider.keys
     }
 
+    @Override
+    Map<String, Entry> asMap() {
+        Map<String, Entry> entryMap = keys.toList().collectEntries {
+            [(it): this[it]]
```

Comment:
this is pretty neat groovy skills, I like it 👍

Reply:
![](https://media.giphy.com/media/IN8gg3Gci335S/giphy.gif)